toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -188,6 +191,11 @@ func (s *SystemConfig) IsValid() (err error) {
 
 	//Validate Encryption
 
+	// Validate locales
+	if len(s.OverrideRpmLocales) > 0 && s.DisableRpmLocales {
+		return fmt.Errorf("invalid [OverrideRpmLocales]: cannot be set when [DisableRpmLocales] is true")
+	}
```

Comment:
What do you think of merging these two configs into one? That way we wouldn't need to add verification logic like the one here and in `customizationmacros.go`.

We replace them with a config like `InstalledRpmLocales` (or `InstalledRPMLocales`, since I'm nit-picky;)) or `InstalledPackagesLocales`. In the docs, in addition to your comment that this works like the RPM `_install_langs` macro, we'd then have to only mention that setting this value to "NONE" would only install the default locale.

Btw., would the value of "en" have the same effect as "NONE"? If so, we wouldn't really need to add any additional comments. Unless we wanted to clarify, how to limit the locales as much as possible.

Reply:
I don't like the idea of tying "disabled" to a user-selectible string, if we realize there is some issue with the implementation we would need to reach out to customers to get them to fix it. If we have a simple `bool` we can hide the implementation details from them.

Reply:
I'm still not convinced on this one:
- With both the override and the boolean we make it easier for the users to make a mistake. And they won't learn about it until later into the build, when this check is triggered.
- It's easy to add the boolean switch, if we learn users have problems with just the override. It's hard to remove a feature once it's published. Or to put it differently: preemptively adding feature we think the users may need can run us into a corner without any clear benefits.

Reply:
Allright, I'll switch.

I still much prefer the explicit boolean though (I feel like adding a half-implemented feature which is unfriendly to use can be just as bad)